### ISSUE TITLE
problems debugging application

### ISSUE THREAD

**Issue by lucamorelli** [aspnet/Tooling#19]
I'm debugging an asp.net 4 application, with an mvc 4 application that calls a routine from a class library. When I move the cursor over the variable I don't see the tooltip of the value, and if I try with Quick Watch I see this.

![untitled](https://cloud.githubusercontent.com/assets/127777/6594314/6cdcd638-c7e0-11e4-88ae-bfc1a6d02a29.png)

**Comment by lucamorelli** [comment]
I just upgraded this application from .net 4 to 4.5. The problem is that with .net System.Net.Http in the project was a reference to the .net 4 version of the dll in the Microsoft..Bcl.* and in the upgrade project the reference wasn't updated... yes, old issue in the migration from .net 4 to 4.5, but I never seen this behavior: compiles without errors but gives this message during debugging.
Now this works, but I have found other problems:

![untitled](https://cloud.githubusercontent.com/assets/127777/6594835/f960d7d6-c7e4-11e4-95e0-f293b55ebc65.png)

trying to click does nothing.
AND

![untitled2](https://cloud.githubusercontent.com/assets/127777/6594857/1c7e2480-c7e5-11e4-869c-c1f7dc7da91c.png)

![image](https://cloud.githubusercontent.com/assets/127777/6594882/32305bb8-c7e5-11e4-94f1-7991741b767f.png)

In short looks like that tooltips or visualizers that are shown placing the cursor above the variables are not working with variables declared inside the foreach cycle, but work with the one decalred before.

**Comment by lucamorelli** [comment]
if I try to use Quick Watch with these variables this is what I see
![untitled2](https://cloud.githubusercontent.com/assets/127777/6594943/bf1caefa-c7e5-11e4-825c-d76d0aa13333.png)

**Comment by sayedihashimi** [comment]
@lucamorelli can you retry with VS2015 and let me know if you are still running into this. Also is it only happening in web projects?

**Comment by sayedihashimi** [comment]
Closing until we get more details whether this is still an issue with the RC release.
